example/many/many.go

```diff
@@ -37,8 +37,26 @@ func main() {
 	var nodes []*network.Node
 	sub := make(chan interface{}, 1000)
 	beacon := "127.0.0.1:10000"
-	log := logger.New()
-	log.SetLevel(logger.Debug)
+
+
+    //add github.com/Microsoft/ApplicationInsights-Go/appinsights and go.uber.org/zap/zapcore to import to be able to run the following commented out code
+	// client := appinsights.NewTelemetryClient("instrumentation_key")
+
+    // logLevelMap := make(map[zapcore.Level]appinsights.SeverityLevel)
+	// logLevelMap[zapcore.DebugLevel] = appinsights.Verbose
+	// logLevelMap[zapcore.InfoLevel] = appinsights.Information
+	// logLevelMap[zapcore.WarnLevel] = appinsights.Warning
+	// logLevelMap[zapcore.ErrorLevel] = appinsights.Error
+	// logLevelMap[zapcore.DPanicLevel] = appinsights.Critical
+	// logLevelMap[zapcore.PanicLevel] = appinsights.Critical
+	// logLevelMap[zapcore.FatalLevel] = appinsights.Critical
+
+	// log, _ := zap.NewDevelopment(zap.Hooks(func(entry zapcore.Entry) error {
+	// 	client.TrackTraceTelemetry(appinsights.NewTraceTelemetry(entry.Message, logLevelMap[entry.Level]))
+	// 	return nil
+	// }))
```

Comment:
This seems cool, but it belongs into a document file (logger.md in a logs folder with this as example code would be nice).